File: py/agc010_a.py
N = int(input())
la = list(map(int, input().split()))

even = 0
odd = 0
for a in la:
    if a%2:
        odd += 1
    else:
        even +=1

if odd % 2  == 0:
    print("YES")
else:
    print("NO")

File: dart/agc010_a.dart
import 'dart:io';
import 'dart:math';

String input() {
  var str = stdin.readLineSync();
  if (str != null) {
    return str;
  } else {
    return '';
  }
}

List<int> inputIntList() {
  return input().split(' ').map(int.parse).toList();
}

void main() {
  int N = int.parse(input());
  List<int> listA = inputIntList();

  int cnt = 0;
  for (int i = 0; i < listA.length; i++) {
    if (listA[i].isOdd) {
      cnt++;
    }
  }
  if (cnt.isOdd) {
    print("NO");
  } else {
    print("YES");
  }
}
